SimplenoteUITests/AllNotes.swift

```diff
@@ -5,29 +5,35 @@ class AllNotes {
     class func openNote(noteName: String) {
         app.tables.cells[noteName].tap()
     }
-    
+
     class func isOpen() -> Bool {
         return app.navigationBars[uidNavBar_AllNotes].exists
     }
 
-    class func open() {        
+    class func open() {
         guard !isOpen() else { return }
-        
+
         app.navigationBars.element.buttons[uidButton_Menu].tap()
         app.tables.staticTexts[uidButton_AllNotes].tap()
     }
 
     class func addNoteTap() {
-        print(">>> Adding new note...")
         app.navigationBars[uidNavBar_AllNotes].buttons[uidButton_NewNote].tap()
     }
 
-    class func createNoteAndLeaveEditor (noteName: String) {
+    class func createNoteAndLeaveEditor(noteName: String) {
+        print(">>> Creating note: " + noteName)
         AllNotes.addNoteTap()
         NoteEditor.clearAndEnterText(enteredValue: noteName)
         NoteEditor.leaveEditor()
     }
 
+    class func createNotes(noteNamesArray: Array<String>) {
```

Comment:
Nitpick. No need to address this, just wanted to point out that, while `Array<String>` is correct, `[String]` is the most common way of writing this.

`[String]` is syntax sugar for `Array<String>`.

Reply:
Thanks! Will update this during next set of changes :-)